src/entities/user/model/user.schema.ts

```diff
@@ -0,0 +1,24 @@
+import {Prop, Schema, SchemaFactory} from '@nestjs/mongoose';
+import {HydratedDocument} from 'mongoose';
+
+export type UserDocument = HydratedDocument<User>;
+
+@Schema()
+export class User {
+  @Prop({required: true})
+  firstName: string;
+
+  @Prop({required: true})
+  lastName: string;
+
+  @Prop({required: true})
+  email: string;
+
+  @Prop({required: true})
+  passwordHash: string;
```

Comment:
`passwordHash` -> `password` . It doesn`t make a sense to rename it to hash, just transform the value but keep the same key.